engine/collection/compliance/core.go

```diff
@@ -323,6 +324,27 @@ func (c *Core) OnBlockVote(originID flow.Identifier, vote *messages.ClusterBlock
 	return nil
 }
 
+func (c *Core) OnTimeoutObject(originID flow.Identifier, timeout *messages.TimeoutObject) error {
+	t := &model.TimeoutObject{
+		View:       timeout.View,
+		HighestQC:  timeout.HighestQC,
+		LastViewTC: timeout.LastViewTC,
+		SignerID:   originID,
+		SigData:    timeout.SigData,
+	}
+
+	c.log.Debug().
+		Hex("replica", originID[:]).
```

Comment:
```suggestion
		Hex("origin_id", originID[:]).
```
Typically use `origin_id` key for this